cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -20,30 +20,30 @@
     # Do not assume that the interpreter is always named like
     # ld-linux-x86_64.so, as on some architectures there can be a version after
     # the .so suffix, eg. ld-linux-aarch64.so.1
-    /{,{,var/lib/snapd/}snap/{snapd,core}/*/}{,usr/}lib{,32,64,x32}/{,@{multiarch}/{,atomics/}}ld{-*,64}.so* mrix,
+    /{,{,var/lib/snapd/}snap/{snapd,core}/*/}{,usr/}lib{,32,64,x32}/{,glibc-hwcaps/{x86-64-v{2,3,4},power{9,10},z1{3,4,5,6}}/,@{multiarch}/{,atomics/}}ld{-*,64}.so* mrix,
```

Comment:
I'm not sure there's a different loader binary built for different hardware capabilities.

Reply:
@bboozzoo Yes, you are probably right, I will remove this line of change.